services/src/main/java/org/apache/druid/cli/CliPeon.java

```diff
@@ -205,7 +223,67 @@ public void configure(Properties properties)
   protected List<? extends Module> getModules()
   {
     return ImmutableList.of(
-        new DruidProcessingModule(),
+        Modules.override(new DruidProcessingModule()).with(
```

Comment:
Rather than doing `Modules.override()`, another option could be to write up a class `TaskQueryProcessingModule extends DruidProcessingModule` (or even just inline it here), where you could just call the super implementation, thus avoiding code duplication.

Reply:
Thanks for the idea!